src/data/markdown/translated-guides/en/01 Get started/02 Installation/01 Troubleshooting.md

```diff
@@ -47,7 +52,24 @@ gpg: keyserver receive failed: No dirmngr
 ```
 
 This happens if it's the first time that the user runs `gpg` , so the directory `/root/.gnupg/` doesn't exist yet.
-To create the directory, run `sudo gpg -k` and try to import the key again.
+
+
+To create the directory, run `sudo gpg -k` and try to import the key again:
+
+```bash
+# list keys to ensure the presence of the `.gnupg` directory
+sudo gpg -k
+
+# delete existing key
+sudo apt-key del k6
```

Comment:
Why is this `apt-key` command mentioned here? This is only relevant for the `apt-key` deprecation error above.

Reply:
https://github.com/grafana/k6-docs/pull/1123/commits/9543e9f47a2e038398946fb550bda5f7bb7d36bb